container-host-files/etc/scf/config/scripts/configure-az.sh

```diff
@@ -36,3 +39,7 @@ else
         export KUBE_AZ="${NODE_AZ}"
     fi
 fi
+
+# Restore outer settings ...
+set +o nounset ;# allow undefined variables
+set +o errexit ;# stop aborting on error
```

Comment:
That means we can never set them in the outer script? (Otherwise, this will now unset them magically, which is probably worse)

Reply:
Do you request the removal of the `nounset` ?

Reply:
Removed `nounset` with commit 4b739f7c4af0a82bd6ee3bd6348edb058a7a0849